groups/instance/cloud-init/templates/artifactory.yml.tpl

```diff
@@ -0,0 +1,15 @@
+#cloud-config
+fqdn: ${instance_fqdn}
+write_files:
+  - path: /opt/jfrog/artifactory/var/etc/system.yaml
+    content: |
+      shared:
+        database:
+          type: postgresql
+          driver: org.postgresql.Driver
+          url: jdbc:postgresql://${db_fqdn}/artifactory
+          username: ${db_username}
+          password: ${db_password}
+runcmd:
```

Comment:
I'd split this content out into a separate `bootstrap-commands.yml.tpl` file as we've done [here](https://github.com/companieshouse/terraform-modules/blob/main/aws/kafka-zookeeper/cloud-init/templates/bootstrap-commands.yml.tpl#L14)